frontend/app.vue

```diff
@@ -0,0 +1,11 @@
+<template>
+  <div>
+    <NuxtRouteAnnouncer />
+    <NuxtWelcome />
+  </div>
+</template>
+<script setup lang="ts">
+import config from '@/data/config.yaml'
+
+console.log(config.project)
```

Comment:
If we keep the console log, I'd rather have it as a `console.debug` instead, to be properly scoped